Codeinwp/neve
# Fix exclusive products

### Summary
Changed the sanitization function for exclusive products. It seems like when comparing in the sanitization function the available categories with the one selected, the selected value was int and the available categories were string.

### Will affect visual aspect of the product
NO

### Test instructions
- Import a starter site with shop and products
- Go to Customizer -> Single Product and add a title and a category
- Test with both neve and neve pro
- Change the products on the shop ( with image effect, wishlist, button position ) and check if they also apply in the slider.
- Test those with 4 products and with more than 4 products


Closes Codeinwp/neve-pro-addon#1486.
